magento/pwa-studio
# Cypress visual testing integration

## Description

Added Cypress Visual Testing plugin. With this, we can write tests that can capture snapshots of the page visually while testing.

## Related Issue
Closes PWA-1661

### Verification Stakeholders
@dpatil-magento 

### Verification Steps
1. Checkout the branch locally.
2. Navigate to the integration tests package.
3. Run `yarn install`
3. Run `yarn test` to start the cypress UI.
4. Run the sample visual test (pageBuilder/homePage.spec.js). Other tests you might see are dummy ones, ignore them for now.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
